core/server/src/main/java/alluxio/master/file/options/DeleteFileOptions.java

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
A lot of unit test should be added, I push this PR just let your guys to discuss whether this pr is meaningful?